runtimes/kotlin-1.6/src/main/kotlin/io/openruntimes/kotlin/RuntimeContext.kt

```diff
@@ -0,0 +1,42 @@
+package io.openruntimes.kotlin
+
+import kotlin.Any
+import kotlin.collections.ArrayList
+import kotlin.collections.arrayListOf
+
+import com.google.gson.GsonBuilder
+import com.google.gson.Gson
+
+class RuntimeContext(req: RuntimeRequest, res: RuntimeResponse) {
+    var req: RuntimeRequest
+    var res: RuntimeResponse
+
+    var logs: ArrayList<String>
+    var errors: ArrayList<String>
+
+    init {
+        this.req = req;
+        this.res = res;
+        this.logs = arrayListOf<String>()
+        this.errors = arrayListOf<String>()
+    }
```

Comment:
Because `req` and `res` are using constructor promotion and `logs` and `errors` are using inline initialization, the `init` block can be removed